requirements.txt

```diff
@@ -66,6 +67,7 @@ oauthlib==2.0.4           # via requests-oauthlib, social-auth-core
 olefile==0.44             # via pillow
 pbr==3.1.1                # via mock
 pdfrw==0.4                # via weasyprint
+phonenumberslite==8.8.5   # via django-phonenumber-field
```

Comment:
We should use `phonenumbers` instead.

Reply:
Actually we should compare their functionality first, maybe we don't use any of the extra features provided by the full library.

Reply:
Well, I put `django-phonenumber-field` and then run `pip-compile --output-file requirements.txt requirements.in` as said in `.in` file.

Reply:
I know. It changes its requirements based on what the project already uses. There are two versions of the `phonenumbers` library. If the full one is not found, the package will pull in the `lite` version.

Reply:
> The phonenumberslite version of the package does not include the geocoding, carrier and timezone metadata (...)
https://github.com/daviddrysdale/python-phonenumbers
I don't think we use those features here.